src/index.html

```diff
@@ -22,6 +22,366 @@
+              <a
+                href="#apple"
+                class="menu-item is-active"
+              >
+                Apple
+              </a>
+            </div>
+          </li>
+          <li>
+            <div id="samsung">
+              <a
+                href="#samsung"
+                class="menu-item"
+              >
+                Samsung
+              </a>
+            </div>
+          </li>
+          <li>
+            <div id="smartphones">
+              <a
+                href="#smartphones"
+                class="menu-item"
+              >
+                Smartphones
+              </a>
+            </div>
+          </li>
+          <li>
+            <div id="laptops">
+              <a
+                href="#laptops"
+                class="menu-item"
+                data-qa="nav-hover"
+              >
+                Laptops & Computers
+              </a>
+            </div>
+          </li>
+          <li>
+            <div id="gadgets">
+              <a
+                href="#gadgets"
+                class="menu-item"
+              >
+                Gadgets
+              </a>
+            </div>
+          </li>
+          <li>
+            <div id="tablets">
+              <a
+                href="#tablets"
+                class="menu-item"
+                >
+                Tablets
+              </a>
+            </div>
+          </li>
+          <li>
+            <div id="photo">
+              <a
+                href="#photo"
+                class="menu-item"
+              >
+                Photo
+              </a>
+            </div>
+          </li>
+          <li>
+            <div id="video">
+              <a
+                href="#video"
+                class="menu-item"
+              >
+                Video
+              </a>
+            </div>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+    <section class="product" data-qa="card">
+      <div class="card">
+        <div class="card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </div>
+        <div class="card__product-code">
+          Product code: 195434
+        </div>
+      </div>
+      <div class="rate">
+        <div class="rate--stars rate--stars-4">
+          <div class="rate--stars__star"></div>
+          <div class="rate--stars__star"></div>
+          <div class="rate--stars__star"></div>
+          <div class="rate--stars__star"></div>
+          <div class="rate--stars__star"></div>
+        </div>
+        <div class="rate--reviews">
+          Reviews: 5
+        </div>
+      </div>
+      <div class="price">
+        <div class="price--label">
+          Price:
+        </div>
+        <div class="price--value">
+          $2,199
+        </div>
+      </div>
+      <button class="to-buy" data-qa="card-hover"> BUY </button>
+    </section>
+
+
+    <section class="product">
+      <div class="card" data-qa="card">
```

Comment:
The `data-qa` attribute is applied to the `card` div. Ensure that this attribute is applied consistently across all card divs if required by the task.